src/components/ScheduleToggle.vue

```diff
@@ -118,7 +116,11 @@ export default {
               class="small-switch"
               color="primary"
               :loading="loading"
-              :disabled="isReadOnlyUser || archived"
+              :disabled="
+                !hasPermission('create', 'run') ||
```

Comment:
If they don't have create run permission but do have delete run they should still be able to turn the schedule off